src/Modules/Admin/Commands.Admin/GetPowerBIEncryptionKeys.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Management.Automation;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    public class GetPowerBIEncryptionKeys : PowerBIClientCmdlet
+    {
+        public const string CmdletName = "PowerBIEncryptionKeys";
```

Comment:
Drop the 's', cmdlet names shouldn't be plural so it should be "PowerBIEncryptionKey"